lib/activity_reporter.rb

```diff
@@ -0,0 +1,43 @@
+# generates text for a report on druid count and logs
+class ActivityReporter
+  DRUID_REGEXP = '[[:lower:]]{2}\\d{3}[[:lower:]]{2}\\d{4}'.freeze
+
+  def output(log_files=default_log_files)
+    today = Time.now.to_date.to_s
+    hash_counter = {}
```

Comment:
This isn't a blocker for me, so I'll just leave it as a comment: what do you think about just naming this `counter`, since you can tell from the right side of the assignment that it's a `Hash`?